src/gromacs/nblib/topology.h

```diff
@@ -70,6 +70,24 @@ std::vector<gmx::ExclusionBlock> toGmxExclusionBlock(const std::vector<std::tupl
 // Add offset to all indices in inBlock
 std::vector<gmx::ExclusionBlock> offsetGmxBlock(std::vector<gmx::ExclusionBlock> inBlock, int offset);
 
+//! Helper class for Topology to keep track of particle IDs
+class EnumerationKey
```

Comment:
I have a slight aesthetic preference that this be called something like LookupTable.

Reply:
How about ParticleSequencer?